service/storage_service.cc

```diff
@@ -3474,19 +3474,6 @@ storage_service::get_range_to_address_map(locator::vnode_effective_replication_m
     co_return co_await construct_range_to_endpoint_map(erm, co_await get_all_ranges(sorted_tokens));
 }
 
-future<> storage_service::handle_state_replacing_update_pending_ranges(mutable_token_metadata_ptr tmptr, inet_address replacing_node, gms::permit_id) {
```

Comment:
@kbr-scylla please cherry-pick and merge this commit

Reply:
Which commit? The comment doesn't show.

Reply:
why? now I probably will have more hassle with conflicts/merge/etc(